src/main/java/Spike.java

```diff
@@ -0,0 +1,217 @@
+import Exceptions.TaskNotFoundException;
+import Tasks.Deadline;
```

Comment:
names representing packages should be in all lowercase, "exceptions.TaskNotFoundException" and "tasks.Deadline"